lib/utils/proxy.ts

```diff
@@ -104,7 +44,7 @@ type ProxyConfig = string | GlobalTunnelNgConfig;
  * 'localhost' and '127.0.0.1' are always excluded. If NO_PROXY is not defined,
  * default exclusion patterns are added for all private IPv4 address ranges.
  */
-async function setupGlobalHttpProxy(settings: CliSettings) {
+export async function setupGlobalHttpProxy(settings: CliSettings) {
```

Comment:
`git` thinks that `app-common.ts` was "renamed" `utils/proxy.ts`, because so much of the content is identical. Only the proxy-specific code was moved to `utils/proxy.ts` though. It was some 150 lines of code, which felt tidier in its own `proxy.ts` file rather than in `app.ts`.